tests/e2e/local/vagrant/Vagrantfile

```diff
@@ -57,7 +57,7 @@ Vagrant.configure("2") do |config|
   #   vb.gui = true
   #
      # Customize the amount of memory on the VM:
-     vb.memory = "8192"
+     vb.memory = "10240"
```

Comment:
This would mean vagrant requires more memory than minikube then..

Reply:
@gargnupur could you give /lgtm ?
It requires a lgtm label. Thanks!